index.html

```diff
@@ -0,0 +1,187 @@
+            @ University of Uyo conference centre.
+          </p>
+        </div>
+      </section>
+
+      <section class="dark-section activities" id="activities">
+       <div>
+        <h2 class="section-title">Main Program</h2>
+
+       
+       </div>
+
+        <ul class="activity-ctn">
+          <li class="activity">
+            <img src="images/program_icon_01 1.svg" alt="Lecture Summit" class="work-image">
+            <h3 class="activity-title">Lecture</h3>
+            <p class="activity-descrip">Listen to the speakers from various universities about the change of curriculum.</p>
+          </li>
+          <li class="activity">
+            <img src="images/program_icon_02 1.png" alt="Exhibition Summit" class="work-image">
+            <h3 class="activity-title">Exhibition</h3>
+            <p class="activity-descrip">Exhibition of different software and apps implementation by students from different universities.</p>
+          </li>
+          <li class="activity">
+            <img src="images/program_icon_03 1.svg" alt="Forum Summit" class="work-image">
+            <h3 class="activity-title">Forum</h3>
+            <p class="activity-descrip">Have the time to share your thoughts and opinions with experts for each software exhibitions.</p>
+          </li>
+          <li class="activity">
+            <img src="images/program_icon_04 1.svg" alt="Workshop Summit" class="work-image">
+            <h3 class="activity-title">Workshop</h3>
+            <p class="activity-descrip">Try learning to develope your own software, rather than just watching.</p>
+          </li>
+          <li class="activity">
+            <img src="images/program_icon_05 1.svg" alt="Biz Expo Summit" class="work-image">
+            <h3 class="activity-title">CC Ignite</h3>
+            <p class="activity-descrip">Get opportunities to affiliate with different students and experts on software issues after the conference.</p>
+          </li>    
+        </ul>
+        <button class="Join-more">ICT for NEEDS conference 2023</button>
+      </section>
+
+      <section class="speaker-section">
+
+     <div class="section-title2">
+      <h2 class="section-title2">Featured Speakers</h2>
+      <img  class="speaker-line"  src="images/title_underline.svg" alt="">
+    </div>
+
+        <div class="speaker-main-cont">
+
+        </div> 
+
+      </section>
+
+      <section class="partner-section" id="sponsor">
+        <h2 class="patner-h2">Partner</h2>
+        <img src="images/title_underline.svg" alt="">
+        <div class="ul-container">
+          <ul class="partner-ul">
+            <li class="partner-li"><img src="images/sponsor_google (1).png" alt=""></li>
+            <li class="partner-li"><img src="images/sponsor_mozilla.png" alt=""></li>
+            <li class="partner-li"><img src="images/sponsor_airbnb.svg" alt=""></li>
+          </ul>
+        </div>
+      </section>
+    </main>
+
+    <footer>
+      <div class="footer-logo">
```

Comment:
- [ ] Kindly fix the footer according to the design on the home page as well as the About page.
- [ ] Your Design
<img width="1440" alt="Screenshot 2023-04-29 at 05 05 18" src="https://user-images.githubusercontent.com/81444338/235301926-95f9b7b5-e16e-48ad-8845-c67aabc83ba9.png">
- [ ] Design Template
<img width="1440" alt="Screenshot 2023-04-29 at 05 06 08" src="https://user-images.githubusercontent.com/81444338/235301944-53e1c7ad-445e-4756-b23f-94e1da46efcc.png">